packages/devtools_app/web/index.html

```diff
@@ -85,7 +85,6 @@
 </head>
 
 <body fullbleed layout vertical>
-    <div id="legacy-dart">
```

Comment:
was removing this intentional? I thought we need this for a recent CL @DaveShuckerow landed?

Reply:
Yes this is intentional. Dave's idea was good but unfortunately our CSS is a bit too fragile to support it. We need to verify that the flutter web version of the app works with this change.
If there is a problem, we will need to do a hybrid solution like tagging every top level div under body with class="legacy-dart" and then removing all those elements.

Reply:
sg 👍